Core/src/org/sleuthkit/autopsy/centralrepository/datamodel/CorrelationAttributeInstance.java

```diff
@@ -276,6 +276,7 @@ public Long getFileObjectId() {
          * @param supported   Is this Type currently supported
          * @param enabled     Is this Type currently enabled.
          */
+        @Messages({"CorrelationAttributeInstance.nullName.message=Database name is null."})
```

Comment:
Is this used?

Reply:
Nope I forgot to actually use it after I made it, will fix that.